### DISCUSSION TITLE
疫情導致外勞短缺，新加坡大量啟用機器人

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#428] [科技新知]
「儘管新加坡大體解除防疫管制，然而預期疫情造成的外勞短缺，仍會持續相當一段時間，這使各行各業越來越傾向改用機器人，一旦改用機器人很可能就「回不去了」，未來新加坡各國外勞疫後逐漸回來找工作時，可能發現原本的工作一部分已永久消失了。」
泡咖啡機器人、書架掃描機器人這些本來高勞力的產業給機器人做沒有不好，疫情狀況下保有人力，成本也降低不少。但文中這段是最令人擔心的，在未來可能會造成失業率大增的反效果。

資料來源：https://technews.tw/2022/06/14/singapores-robot-workforce-plugs-labour-gaps/

**Comment by YuHengChen54** [comment]
我記得日本會一直發展機器人就是因為他們不喜歡雇用外籍移工，所以才會寧願很多工作都讓機器人替代，也不願用外籍移工。

**Comment by BingRong1217** [comment]
機器人代替人力後的失業率確實蠻可觀的

**Comment by Ben6uo** [comment]
新加坡薪水福利高阿 我想去當外勞
